Code/Tools/FontImporter/FontImporter.cpp

```diff
@@ -0,0 +1,382 @@
+
+        baselineOffset = ezMath::Max(baselineOffset, (ezInt32)(slot->metrics.horiBearingY >> 6));
+        lineHeight = ezMath::Max(lineHeight, fontGlyph.m_Height);
+
+        // Load kerning and store char
+        if (!isMissingGlyph)
+        {
+          FT_Vector resultKerning;
+          for (ezUInt32 charIndexRangeIter = 0; charIndexRangeIter < importOptions.m_charIndexRanges.GetCount(); charIndexRangeIter++)
+          {
+            auto kerningIter = importOptions.m_charIndexRanges[charIndexRangeIter];
+            for (ezUInt32 kerningCharIdx = kerningIter.m_Start; kerningCharIdx <= kerningIter.m_End; kerningCharIdx++)
+            {
+              if (kerningCharIdx == characterIndex)
+                continue;
+
+              error = FT_Get_Kerning(face, characterIndex, kerningCharIdx, FT_KERNING_DEFAULT, &resultKerning);
+              if (error)
+              {
+                ezLog::Error("Failed to get kerning information for character: {0}", characterIndex);
+                return EZ_FAILURE;
+              }
+
+              ezInt32 kerningX = (ezInt32)(resultKerning.x >> 6); // Y kerning is ignored because it is so rare
+              if (kerningX == 0)                                  // We don't store 0 kerning, this is assumed default
+                continue;
+
+              ezKerningPair pair;
+              pair.m_Amount = kerningX;
+              pair.m_OtherCharacterId = kerningCharIdx;
+
+              fontGlyph.m_KerningPairs.PushBack(pair);
+            }
+          }
+
+          rawFontBitmap.m_Characters[characterIndex] = fontGlyph;
+        }
+        else
+        {
+          rawFontBitmap.m_MissingGlyph = fontGlyph;
+        }
+      }
+
+      ezImageHeader imgHeader;
+      imgHeader.SetWidth(page.Width);
+      imgHeader.SetHeight(page.Height);
+      imgHeader.SetImageFormat(ezImageFormat::Enum::R8_UNORM);
+
+      ezImage& outImg = rawFontBitmap.m_Textures.ExpandAndGetRef();
+      outImg.ResetAndAlloc(imgHeader);
+
+      auto pixelData = outImg.GetBlobPtr<ezUInt8>();
+      ezMemoryUtils::ZeroFill(pixelData.GetPtr(), pixelData.GetCount());
+      ezMemoryUtils::Copy(pixelData.GetPtr(), pixelBuffer, bufferSize);
+
+      {
+        ezStringBuilder fontPageName;
+        fontPageName.Format("{0}_{1}_{2}", fontFileName, fontSize, pageIndex);
+
+        ezStringBuilder fontPagePath(inputFile);
+        fontPagePath.Append(fontPageName.GetData());
+        ezStringBuilder fontImagePath(fontPagePath);
+        fontImagePath.Append(".png");
+
+        ezStringBuilder fontImageAbsPath;
+
+        ezResult resolved = ezFileSystem::ResolvePath(fontImagePath, &fontImageAbsPath, nullptr);
+
+        ezFileWriter file;
+
+        file.Open(fontImageAbsPath);
+
+        auto ezResult = ezImageFileFormat::GetWriterFormat("png")->WriteImage(file, outImg, ezLog::GetThreadLocalLogSystem(), "png");
+
+        EZ_ASSERT_ALWAYS(ezResult == EZ_SUCCESS, "Failed to write file.");
+
+        file.Close();
+      }
```

Comment:
Maybe add a command line switch for this feature, so that you can keep it in, but disable it by default later.